src/test/java/org/folio/rest/api/LoansApiTest.java

```diff
@@ -1342,6 +1342,22 @@ public void canSearchByLoanStatus() throws Exception {
     assertThat(openLoans, hasItem(openLoan.getId()));
   }
 
+  @Test
+  public void canSearchByLoanItemStatus() throws Exception {
+    final IndividualResource agedToLostLoan = loansClient.create(
+      new LoanRequestBuilder().agedToLost());
+
+    loansClient.create(new LoanRequestBuilder().lostAndPaid());
+
+    final List<String> agedToLostLoans = loansClient.getMany("itemStatus==Aged to lost")
```

Comment:
Why does the builder define a loan that is `lost and paid` and yet the query tries to find a loan where the item is `aged to lost`?

Does `lostAndPaid` specify more than one aspect of the loan?

Reply:
there is one more loan declaration above the lostAndPaidLoan.